app/Dockerfile

```diff
@@ -29,7 +39,7 @@ COPY --from=deps /code/node_modules ./node_modules
 COPY --from=deps /code/app/node_modules ./app/node_modules
 COPY --from=deps /code/client-libs/typescript/node_modules ./client-libs/typescript/node_modules
 COPY . .
-RUN cd app && SKIP_ENV_VALIDATION=1 pnpm build
+# RUN cd app && SKIP_ENV_VALIDATION=1 pnpm build
```

Comment:
Why did you comment out the build step?

Reply:
Accidentally committed this, was just to make local testing faster. Added back.